src/http/server.ts

```diff
@@ -565,6 +565,47 @@ export class Server {
           ),
         };
 
+        newPaths.push(pathObj);
+      } else if (path.includes("?") === true) { // optional params
+        let tmpPath = path;
+        // Replace required params
+        const numberOfRequiredParams = path.split("/").filter((param) => {
+          return (param.includes(":") || param.includes("{")) &&
```

Comment:
can we turn this into a function or add a comment above this por favor?

Reply:
Would it best be suited to add it as a private method to server or resource class? cause it's for the server class, but mostly related to resources - or maybe the http request service?

Reply:
a comment is fine

Reply:
Felt it could do with some extra comments anyway amigo (on top of previous commit): https://github.com/drashland/deno-drash/pull/337/commits/9e4325bac38bc4c14857a79e48f540aedf664991

Reply:
nah we good